common/importer/importer_constants.h

```diff
@@ -0,0 +1,13 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMMON_IMPORTER_IMPORTER_CONSTANTS_H_
+#define BRAVE_COMMON_IMPORTER_IMPORTER_CONSTANTS_H_
+
+// Pref file that holds installed extension list.
+extern const char kChromeExtensionsPreferencesFile[];
+extern const char kChromeExtensionsListPath[];
```

Comment:
you can just use `constexpr char kChromeExtensionsPreferencesFile[] = "...";` and avoid adding .cc